app/commands/webhooks/process_membership_update.rb

```diff
@@ -0,0 +1,27 @@
+module Webhooks
+  class ProcessMembershipUpdate
+    include Mandate
+
+    initialize_with :action, :username, :organization
+
+    def call
+      return unless organization == 'exercism'
```

Comment:
This is an extra sanity check. It should not be possible to get here due to us validating the signature of the webhook call, which is only correct if people have access to the secret that we use. If they somehow _do_ get access to the secret, they still wouldn't be able to add a user from another organization.

Reply:
Nice. I like this thinking!